### PR TITLE
Update buy_sell_stock.py

### PR DESCRIPTION
**Pull request by dong-jy**
I don't know exactly if I was correct.
Please review it.
Thanks.

### CODE REVIEW

File: dp/buy_sell_stock.py [hunk 1]
```diff
@@ -23,12 +23,13 @@
 def max_profit(prices):
     """
     input: [7, 1, 5, 3, 6, 4]
-    diff : [X,-6, 5,-2, 3,-2]
+    diff : [X,-6, 4,-2, 3,-2]
```
Comment (ankit167) [inline dp/buy_sell_stock.py]: Please use spaces around ','

File: dp/buy_sell_stock.py [hunk 2]
```diff
@@ -23,12 +23,13 @@
...
     :type prices: List[int]
     :rtype: int
     """
     cur_max, max_so_far = 0, 0
     for i in range(1, len(prices)):
```
Comment (ankit167) [inline dp/buy_sell_stock.py]: should 'i' start from 0 and 'j' start from 'i+1'? Correct me if I am wrong.

**Comment by ankit167** [comment]
@dong-jy Since, we are allowed to make only a single transaction, this problem is same as finding max difference between two numbers, such that the larger number appears after the smaller number. Your solution is correct. However, it takes O(n^2) time complexity. I believe, we can work this problem out in O(n) time. In case you are interested, please append existing code with the optimized version as well.

File: dp/buy_sell_stock.py (hunk 1 above)
Comment (dong-jy) [inline dp/buy_sell_stock.py]: Thanks!
But what does 'nit' mean?
And I used space around ',', kept same format with original lines.

File: dp/buy_sell_stock.py (hunk 2 above)
Comment (dong-jy) [inline dp/buy_sell_stock.py]: Got it.

File: dp/buy_sell_stock.py (hunk 1 above)
Comment (ankit167) [inline dp/buy_sell_stock.py]: nit is the short form for 'nit pick'. It denotes a change which is not _important_, but is technically _correct_

I can see space missing before the negative numbers. (the reason I commented)

Comment (dong-jy) [inline dp/buy_sell_stock.py]: @ankit167 
 I just revised the code according to your review.
May you review it again if your are free? Or just edit or delete this PR is if doesn't meet the standard.

The original author just  wrote this way in comment (without space before minus numbers), so I follow him.

**Comment by ankit167** [comment]
@dong-jy May be, we should also add algorithm to find the maximum profit, where multiple transactions (buy/sell) are allowed. That can be a separate pull request. Please take this one up as well. Thanks.

**Pull request merged**